BunqSdk/Exception/ExceptionFactory.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+
+namespace Bunq.Sdk.Exception
+{
+    public class ExceptionFactory
+    {
+        private const int HTTP_RESPONSE_CODE_BAD_REQUEST = 400;
```

Comment:
This constant group can have a header saying something like "Error response codes"